packages/editor/src/lib/config/TLUserPreferences.ts

```diff
@@ -118,7 +119,7 @@ export const defaultUserPreferences = Object.freeze({
 	name: 'New User',
 	locale: getDefaultTranslationLocale(),
 	color: getRandomColor(),
-	isDarkMode: false,
+	colorScheme: 'light',
```

Comment:
```suggestion
```

i think the default should be `undefined` here in order to allow `inferDarkMode` to be respected by default in the user preferences?